hazelcast/src/test/java/com/hazelcast/internal/ascii/MemcacheTest.java

```diff
@@ -57,11 +58,17 @@
     final static Config config = new XmlConfigBuilder().build();
 
     @BeforeClass
+    public static void setup() throws IOException {
+        System.setProperty(GroupProperty.MEMCACHE_ENABLED.getName(),"true");
```

Comment:
It will probably not affect the other tests but it is better to set the property on config

config.setProperty(GroupProperty.MEMCACHE_ENABLED.getName(),"true")